src/core/sign.ml

```diff
@@ -64,12 +66,15 @@ let current_sign () =
   in
   PathMap.find mp !loaded
 
+let create_sym : string -> term -> prop -> expo -> sym = fun name typ p e ->
```

Comment:
Thanks to remove.

Reply:
On the other hand, it would be useful to add a function similar to add_symbol but for inductive:
```
add_inductive : t -> sym -> sym list -> sym -> unit
= fun sign typ ind_cons ind_prop ->
let ind = { ind_cons; ind_prop } in
sign.sign_ind := SymMap.add typ ind sign.sign_ind
```

Reply:
This creates a "Dependency cycle" due to t in the signature.

Reply:
Done